src/views/pages/profile/component/ElementType/ElementInfo.tsx

```diff
@@ -0,0 +1,15 @@
+import { CharacterTypeWrapper, ElementName, ElementImage } from './styled';
+import { Element } from 'views/types/profilePage';
+
+interface ElementTypeProps {
+  elementData: Element;
+}
+
+export const CharacterType: React.FC<ElementTypeProps> = ({ elementData }) => {
+  return (
+    <CharacterTypeWrapper>
```

Comment:
check if we can use the same component we use for login buttons. They seems similar